docs/examples.rst

```diff
@@ -183,42 +271,203 @@ infrastructure defined above, and also define an
+perform annual averages, both for each gridpoint and regionally
+averaged.
+
+Results
+=======
+
+The result is a list of :py:class:`aospy.Calc` objects, one per
+simulation.
 
 .. ipython:: python
 
-   calc_int = CalcInterface(
-       proj=example_proj,
-       model=example_model,
-       run=example_run,
-       var=precip,
-       date_range=(datetime(4, 1, 1), datetime(6, 12, 31)),
-       intvl_in='monthly',
-       dtype_in_time='ts',
-       intvl_out='ann',
-       dtype_out_time='reg.av',
-       region={'globe': globe}
-   )
-   Calc(calc_int).compute()
+    calcs
+
+Each :py:class:`aospy.Calc` object includes the paths to the output
+and the results of each output type.
+
+.. ipython:: python
+
+    calcs[0].path_out
+    calcs[0].data_out
+
+**(S. Hill: Still need to finish up everything below this point)**
+
+.. note::
+
+   You may have noticed that ``subset_`` and ``raw_`` coordinates have
+   years 1678 and later, when our data was from model years 4
+   through 6.  This is because technical details upstream (in numpy)
+   limit the range of supported years to roughly 1677 to 2234.
 
-This produces a new file, located in:
+   As a workaround, aospy pretends that any timeseries that starts
+   before the beginning of this range actually starts at the start of
+   this range. A solution to this problem at the xarray level is
+   currently under way, at which point all meaningful dates will be
+   supported without any workarounds or date range limits.
+
+Let's plot the time average at each gridcell of all four variables:
 
 .. ipython:: python
 
-   calc_int.path_out['reg.av']
+    from matplotlib import pyplot as plt
+
+    fig = plt.figure()
+
+    for i in range(4):
+        ax = fig.add_subplot(2, 2, i+1)
+        xr.open_dataset(calcs[i].path_out['av']).to_array().plot(ax=ax)
 
-We find that the global annual mean total precipitation rate for this
-run (converting to units of mm per day) is:
+    @savefig plot_av.png width=100%
+    plt.show()
+
+We see that the convective precipitation dominates the total in the
+Tropics, but moving poleward the gridscale condensation plays an
+increasingly larger fractional role. [#ls-conv]_
+
+Now let's examine the regional averages.  We find that the global
+annual mean total precipitation rate for this run (converting to units
+of mm per day) is:
 
 .. ipython:: python
 
-    xr.open_dataset(calc_int.path_out['reg.av']).globe * 86400.
+    calcs[0].data_out['reg.av'].to_array()*86400
+
+As was evident from the plots, we see that the global mean
+precipitation is dominated by the convective rainfall, and the
+tropical mean even more so.
+
+Beyond this simple example
+--------------------------
+
+We can re-use our object library at will to perform new calculations
+or re-compute old ones.  We can also add new objects.  For example,
+suppose we performed a new simulation in which we increased the
+surface albedo or introduced a rectangular continent.  All we would
```

Comment:
In this case of a rectangular continent, I might say "not so fast!" :)

Currently the safest thing to do in that situation would be to create a new Model with a different land mask (to make sure regional calculations that depended on land would be accurate).  Changing surface albedo is fine for the same Model, however.  

If you want some other parameters to tweak as an example (while keeping the same Model object) you could say increase the solar constant or turn off the convection scheme.